build/jobs/cleanup-aks.yml

```diff
@@ -0,0 +1,29 @@
+
+parameters:
+- name: deploymentEnvironmentName
```

Comment:
Nit: We haven't been passing deployment environment name as a parameter because the global variable DeploymentEnvironmentName can be used.